rosmon_core/src/main.cpp

```diff
@@ -75,6 +75,9 @@ void usage()
 		"                 Use GROUP as name of the launch group. By default, empty\n"
 		"  --launch-config=CONFIG\n"
 		"                 Use CONFIG as name of the launch config. By default, empty\n"
+                "  --respawn-all=BOOL\n"
```

Comment:
can you do this similar to https://github.com/xqms/rosmon/pull/109,
basically have `--respawn-attr="obey|force_true|force_false"